Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -127,13 +123,23 @@ function Main() {
 		cd
 
 		# Append the test log to the main log files.
-		cat /usr/src/linux/TestExecution.log >> TestExecution.log
-		cat /usr/src/linux/TestExecutionError.log >> TestExecutionError.log
+		if [ -f /usr/src/linux/TestExecution.log ]; then
+			cat /usr/src/linux/TestExecution.log >> TestExecution.log
+		fi
+		if [ -f /usr/src/linux/TestExecutionError.log ]; then
+			cat /usr/src/linux/TestExecutionError.log >> TestExecutionError.log
+		fi
 	fi
 
 	if [[ "$DISTRO" =~ "redhat" ]];then
-		sed -i -e "s/rootdelay=300/rootdelay=300 resume=$sw_uuid/g" /etc/default/grub
-		LogMsg "$?: Updated the /etc/default/grub with resume=$sw_uuid"
+		_entry=$(cat /etc/default/grub | grep 'rootdelay=')
+		if [ $_entry ]; then
+			sed -i -e "s/rootdelay=300/rootdelay=300 resume=$sw_uuid/g" /etc/default/grub
+			LogMsg "$?: Updated the /etc/default/grub with resume=$sw_uuid"
+		else
+			echo 'rootdelay=300 resume=$sw_uuid' >> /etc/default/grub
```

Comment:
simple append will not work, it needs add in GRUB_CMDLINE_LINUX_DEFAULT

Reply:
Fixed in db71c5c1ca66ae210a70117bba412d4ea18adb73. I did not pay attention here because I am working on Analysis-K work now. :)